Serialize/Services.cs

```diff
@@ -1128,7 +1166,7 @@ public static void EncodeEventNotifyUnconfirmed(EncodeBuffer buffer, BacnetEvent
             EncodeEventNotifyData(buffer, data);
         }
 
-        public static void EncodeAlarmSummary(EncodeBuffer buffer, BacnetObjectId objectIdentifier, uint alarmState, BacnetBitString acknowledgedTransitions)
+		public static void EncodeAlarmSummary(EncodeBuffer buffer, BacnetObjectId objectIdentifier, uint alarmState, BacnetBitString acknowledgedTransitions)
```

Comment:
revert unintended code formatting change please

Reply:
it seems fixed now...